vowpalwabbit/core/src/reductions/details/automl/automl_oracle.cc

```diff
@@ -178,64 +223,17 @@ void config_oracle<oracle_impl>::insert_config(set_ns_list_t&& new_elements,
   float priority = (*calc_priority)(configs[valid_config_size], ns_counter);
   index_queue.push(std::make_pair(priority, valid_config_size));
   ++valid_config_size;
+  return true;
 }
 
-// This will generate configs based on the current champ. These configs will be
```

Comment:
Can we keep/update this comment where needed?

Reply:
done